packages/features/ee/workflows/components/WorkflowListPage.tsx

```diff
@@ -1,108 +1,169 @@
+import { Workflow, WorkflowStep } from "@prisma/client";
```

Comment:
All diffs in this page are because the v1 version had the same name, and I moved v2 to the same place. So this is actually the difference between v1 and v2.